curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.html

```diff
@@ -0,0 +1,113 @@
+<html>
+<head>
+<meta charset="utf-8">
+<title>CVdrawing</title>
+</head>
+<body>
+<p id="status">OpenCV.js is loading...</p>
+<canvas id="canvas" width="640" height="426">
+</canvas>
+<img id="src" src="bird.jpg" style="display:none"/>
+<input type="button" id="myButton" value="Test" disabled=true
+onclick="inpaint()"/>
+
+<script type="text/javascript">
+
+var checker = false;
+var src;
+var srcmat;
+var dst;
+var dstmat;
```

Comment:
Let's also try to have a single `src` mat: we do
```js
function init()
{
  var rgba = cv.imread("src");
  cv.cvtColor(rgba, src, cv.COLOR_RGBA2RGB);
```

Reply:
srcmat is no more global, it is initialized in "inpaint" function, used there, and deleted there.